src/main/java/org/springframework/data/elasticsearch/core/MappingBuilder.java

```diff
@@ -404,6 +409,9 @@ private void addFieldMappingParameters(XContentBuilder builder, Object annotatio
 		if (copyTo != null && copyTo.length > 0) {
 			builder.field(FIELD_COPY_TO, copyTo);
 		}
+		if (ignoreAbove > -1) {
```

Comment:
```suggestion
		if (ignoreAbove != null) {
```
after changing to `Integer`